hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcServer.java

```diff
@@ -182,21 +176,4 @@ public int getNumOpenConnections() {
     // allChannels also contains the server channel, so exclude that from the count.
     return channelsCount > 0 ? channelsCount - 1 : channelsCount;
   }
-
-  @Override
-  public Pair<Message, CellScanner> call(BlockingService service,
```

Comment:
This is related to this issue?

Reply:
Oh, this should have already been landed on master. Let's check if we need a rebase.